# Bisulfilter
Tool for filtering BSSeeker2 [1] methylation calls obtained by whole genome bisulfite sequencing.

More specifically, this Python3 tool filters **CGmap** files generated by BSSeeker2 according to criteria
specified by the user. Alternatively, it can also summarize the CG map file
based on these criteria

If you use this tool, please cite:
[B Cuypers, F Dumetz, P Meysman, K Laukens, G De Muylder, J-C Dujardin, MA Domagalska
The absence of C-5 DNA methylation in Leishmania donovani allows DNA enrichment from complex samples](https://doi.org/10.1101/747063)

```
Usage: python Bisulfilter.py [-h] [-s] [-c INT] [-m FLOAT] CGmap

positional arguments:
  CGmap       input CGmap

optional arguments:
  -h, --help  show this help message and exit
  -s          show summary instead
  -c INT      minimal number of reads covering the postion for it to be
              reported (default: 25)
  -m FLOAT    proportion of reads that need to be methylated for a specific
              position for it to be reported (default: 0.8)
```

For all questions, problems and suggestions please contact the author Bart
Cuypers at bart.cuypers@uantwerpen.be


[1] [1.	Guo W, Fiziev P, Yan W, Cokus S, Sun X, Zhang MQ, Chen P-Y, Pellegrini M: BS-Seeker2: a versatile aligning pipeline for bisulfite sequencing data. BMC genomics 2013, 14(1):774.](https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-14-774)
